PersonSearchDetails/PageObjects/Person/PersonMethods.cs

```diff
@@ -4,38 +4,35 @@
 using OpenQA.Selenium.Support.UI;
 using System;
 using System.Linq;
+using WCCIS.Specs.PageObjects;
 using WCCIS.Specs.Extentions;
 
 namespace PersonSearchDetails.PageObjects
 {
     internal class PersonMethods
         // a collection of methods that can be used in the Person area of CareDirector
     {
+
+
+
+        // THIS DOES NOT FOLLOW THE PAGE OBJECT PATTERN - WE MAY NEED TO RECONSIDER WHERE THIS FILE SHOULD LIVE
+
+
         public static string dateMovedInNew = "01/01/2003";
         public string lastName { get; set; }
 
         // this method completes the mandatory fields required to add a valid address to a Person
         public static void enterAddressDetails(Browser xrmBrowser, IWebDriver driver, string propertyNo, string firstLineOfAddress, string townCity, string county, string postCode)
         {
-            driver.FindElement(By.XPath("//*[@id=\"cw_propertyno_cl\"]")).Click();
-            driver.FindElement(By.XPath("//*[@id=\"cw_propertyno_i\"]")).SendKeys(propertyNo);
-            driver.FindElement(By.XPath("//*[@id=\"cw_propertyno_i\"]")).SendKeys(Keys.Return);
-            xrmBrowser.ThinkTime(1000);
-            driver.FindElement(By.XPath("//*[@id=\"address1_line1_cl_span\"]")).Click();
-            driver.FindElement(By.XPath("//*[@id=\"address1_line1_i\"]")).SendKeys(firstLineOfAddress);
-            driver.FindElement(By.XPath("//*[@id=\"address1_line1_i\"]")).SendKeys(Keys.Return);
-            xrmBrowser.ThinkTime(1000);
-            driver.FindElement(By.XPath("//*[@id=\"address1_city_cl\"]")).Click();
-            driver.FindElement(By.XPath("//*[@id=\"address1_city_i\"]")).SendKeys(townCity);
-            driver.FindElement(By.XPath("//*[@id=\"address1_city_i\"]")).SendKeys(Keys.Return);
-            xrmBrowser.ThinkTime(1000);
-            driver.FindElement(By.XPath("//*[@id=\"address1_stateorprovince_cl\"]")).Click();
-            driver.FindElement(By.XPath("//*[@id=\"address1_stateorprovince_i\"]")).SendKeys(county);
-            driver.FindElement(By.XPath("//*[@id=\"address1_stateorprovince_i\"]")).SendKeys(Keys.Return);
-            xrmBrowser.ThinkTime(1000);
-            driver.FindElement(By.XPath("//*[@id=\"address1_postalcode_cl\"]")).Click();
-            driver.FindElement(By.XPath("//*[@id=\"address1_postalcode_i\"]")).SendKeys(postCode);
-            driver.FindElement(By.XPath("//*[@id=\"address1_postalcode_i\"]")).SendKeys(Keys.Return);
+
+            throw new NotImplementedException("");
```

Comment:
Not an issue but just wondering what happened to this code, looks like its been added but cant be found anywhere in my local repo on your branch

Reply:
The exception was unfinished - I've now amended it to have some text. The overall method has been extracted into individual test step methods, so I've added the exception to show this is no longer relevant. 

We should delete the method (and possibly the class) in a future clean up.